ompi/mpi/c/generate_bindings.py

```diff
@@ -0,0 +1,1278 @@
+
+        prefix = indent_count * tab
+        new_lines.append(f'{prefix}{line}')
+
+        # Opening bracket
+        if '{' in line:
+            indent_count += 1
+    return new_lines
+
+
+def standard_abi(base_name, template):
+    """Generate the standard ABI functions."""
+    template.print_header()
+    print(f'#include "{ABI_INTERNAL_HEADER}"')
+
+    # Static internal function (add a random component to avoid conflicts)
+    internal_name = f'ompi_abi_{template.prototype.name}'
+    internal_sig = template.prototype.signature('ompi', internal_name,
+                                                count_type='MPI_Count')
+    print(INLINE_ATTRS, internal_sig)
+    template.print_body(func_name=base_name)
+
+    def generate_function(prototype, fn_name, internal_fn, count_type='int'):
+        """Generate a function for the standard ABI."""
+        print_profiling_header(fn_name)
+
+        # Handle type conversions and arguments
+        params = [param.construct('standard') for param in prototype.params]
+        print(prototype.signature('standard', fn_name, count_type=count_type))
+        print('{')
+        lines = []
+        return_type = prototype.return_type.construct('standard')
+        lines.append(f'{return_type.tmp_type_text()} ret_value;')
+        for param in params:
+            if param.init_code:
+                lines.extend(param.init_code)
+        pass_args = ', '.join(param.argument for param in params)
+        lines.append(f'ret_value = {internal_fn}({pass_args});')
+        for param in params:
+            if param.final_code:
+                lines.extend(param.final_code)
+        lines.extend(return_type.return_code('ret_value'))
+
+        # Indent the lines
+        lines = indent_lines(lines, 4 * ' ', start=1)
+        for line in lines:
+            print(line)
+        print('}')
+
+    generate_function(template.prototype, base_name, internal_name)
+    if template.prototype.need_bigcount:
+        base_name_c = f'{base_name}_c'
+        generate_function(template.prototype, base_name_c, internal_name,
+                          count_type='MPI_Count')
+
+
+def gen_header(args):
+    """Generate an ABI header and conversion code."""
+    prototypes = [SourceTemplate.load(file_, args.srcdir).prototype for file_ in args.file]
+
+    builder = ABIHeaderBuilder(prototypes, external=args.external)
+    builder.dump_header()
+
+
+def gen_source(args):
+    """Generate source file."""
+    template = SourceTemplate.load(args.source_file)
+
+    base_name = mpi_fn_name_from_base_fn_name(template.prototype.name)
+    if args.type == 'ompi':
+        ompi_abi(base_name, template)
+    else:
+        standard_abi(base_name, template)
+
+
+def main():
+    if len(sys.argv) < 2:
+        # Fix required for Python 3.6
+        print('ERROR: missing subparser argument (see --help)')
+        sys.exit(1)
+
+    parser = argparse.ArgumentParser(description='generate ABI header file and conversion code')
```

Comment:
Sorry I should have been more clear with my commit message. I removed all ABI-specific changes from the make-system, but the script still includes the unused ABI code. This seemed the best route to take, instead of completely removing the extra ABI code, which is not affecting how ompi is being built right now. I can remove it though.